source/2_universal_payload.rst

```diff
@@ -3,6 +3,10 @@
 Universal Payload
 ===================
 
+July 30, 2024
+
+A new universal payload specification is published at https://universalpayload.github.io/spec/index.html. USF universal payload is going to adopt this new universal payload specification.
```

Comment:
i think this would be enough? no need the index.html :)
https://universalpayload.github.io/spec/

Reply:
yes. no need the index.html

Reply:
Thanks Dong Guo! However I m not sure why I can’t mark this comment as resolved…